projects/core/src/routing/configurable-routes/config/default-config.ts

```diff
@@ -0,0 +1,6 @@
+import { defaultRoutesConfig } from '../routes-config';
+import { ConfigurableRoutesConfig } from './configurable-routes-config';
+
+export const defaultConfigurableRoutesConfig: ConfigurableRoutesConfig = {
```

Comment:
Super minor thing: exported symbol should match file name (to be in line with angular code guideliness), so for this case it should be `default-configurable-routes-config.ts`